charts/observability-pipelines-worker/templates/_pod.tpl

```diff
@@ -91,6 +91,17 @@ containers:
     volumeMounts:
       - name: data
         mountPath: "{{ .Values.datadog.dataDir | default "/var/lib/observability-pipelines-worker" }}"
+{{- if or .Values.bootstrapBackend.secretFile.enabled .Values.bootstrapBackend.custom.enabled }}
+      - name: opworker-bootstrap-defaults
+        mountPath: /etc/observability-pipelines-worker
+        readOnly: true
+{{- end }}
+{{- if eq (include "opw.bootstrapMode" .) "secretFile" }}
+      - name: secret-file-backend
+        mountPath: /etc/observability-pipelines-secrets/secrets.json
```

Comment:
```suggestion
        mountPath: {{ .Values.bootstrapBackend.secretFile.mountPath }}
```
should we use the variable here so that it mounts to the right place if the var changes?